Microsoft.Toolkit.Parsers/Markdown/MarkdownDocument.cs

```diff
@@ -372,15 +387,37 @@ public LinkReferenceBlock LookUpReference(string id)
                 return null;
             }
 
-            LinkReferenceBlock result;
-            if (_references.TryGetValue(id, out result))
+            if (_references.TryGetValue(id, out LinkReferenceBlock result))
             {
                 return result;
             }
 
             return null;
         }
 
+        /// <summary>
+        /// Adds Custom Schemes to the list of existing schemes
```

Comment:
This method also clears any custom schemes that might have been added previously. Since the `KnownSchemes` property is already public and the developer can manage it - I still don't think this method is necessary.

Reply:
I apologize, I'm not sure I understand. Why can't you access the KnownSchemes static property which is already public?